typing/value_rec_check.ml

```diff
@@ -273,7 +273,9 @@ let classify_expression : Typedtree.expression -> sd =
     | Texp_assert _
     | Texp_try _
     | Texp_override _
-    | Texp_letop _ ->
+    | Texp_letop _
+    | Texp_quotation _
```

Comment:
Intuitively, they feel like static?

Reply:
Hm, you might be right about this. @lpw25 thoughts?